cmake/globals.cmake

```diff
@@ -36,6 +36,9 @@ elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64")
 elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "arm64")
   # Apple Silicon
   set(TARGET_PROCESSOR "aarch64")
+elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "ARM64")
+  # Windows on Arm
+  set(TARGET_PROCESSOR "aarch64")
```

Comment:
These only differ by case is there a better way to handle this?

Reply:
In general, I prefer to use explicit code for something like this, but it's possible to use [TOLOWER](https://cmake.org/cmake/help/latest/command/string.html#tolower) function instead.